### ISSUE TITLE
Issues with inline comments

### ISSUE THREAD

**Issue by claudiuapetrei** [kkinnear/zprint-std#1]
Ex:
```
;; before
(foo bar
     baz             ;; <-- 2nd line is "first-arg aligned"
  qux)

;; after
(foo bar
  baz             ;; <-- 2nd line is "first-arg aligned"  qux)

;; expected
(foo bar
  baz             ;; <-- 2nd line is "first-arg aligned"  
  qux)

**Comment by kkinnear** [comment]
Yes, I forgot to tie off the normal zprint comment handling, and it interacts badly with the code I added to try out this formatting approach.

At first look, this seems like a reasonable workaround.  Use this options map:

`./zprintm-std-0.1.1 '{:comment {:inline? false :wrap? false}}' <infile.clj >outfile.clj`

If you don't want to put this on the command line, you could put it in your `~/.zprintrc`.  All of the normal zprint config is run with this zprint-std thing.
